velox/functions/prestosql/StringFunctions.cpp

```diff
@@ -290,7 +290,8 @@ class ConcatFunction : public exec::VectorFunction {
  * If search is an empty string, inserts replace in front of every character
  *and at the end of the string.
  **/
-class Replace : public exec::VectorFunction {
+template <bool ignoreEmptyReplaced>
```

Comment:
This `ignoreEmptyReplaced` is only used in SparkSql, can we move the implementation to Spark? CC @mbasmanova

Reply:
You're right, the `ignoreEmptyReplaced` implementation is inappropriate in the presto directory.

Spark and presto have roughly the same `replace` function, there is just a difference when `replaced` is empty. If we copy the `Replace` class into the spark directory, a lot of code will be duplicated and perhaps this will make the code less maintainable.

Maybe we should move the `ReplaceBase` class? I don't know if there is a better way, do you have any suggestions? Thanks!